### ISSUE TITLE
Can't tap on annotation while annotation tool is active

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-android#175]
1. Select the Underline tool
2. Select some text
3. Tap on the annotation

On iOS, this opens the annotation widget, but on Android it doesn't do anything. You have to deselect the tool before you can tap on the annotation.

**Comment by Dima-Android** [comment]
I’ve looked into it and it seems there is a difference between the way PSPDFKIT operates on iOS and Android:
On iOS after the “enterAnnotationCreationMode(annotationTool)” method was called OnAnnotationSelectedListener is getting called when annotation is getting clicked on.

On Android in similar conditions this listener is just not getting called. Not only that, even the onPageClick is not getting called.
In other words we don’t even get info on which point on the document the user clicked, let alone annotation. Only after exitCurrentlyActiveMode is called and we are no longer in annotation editing mode those callbacks are getting called.

**Comment by dstillman** [comment]
Can we report this to them? This is going to make the latched annotation modes fairly useless — if you use comments at all, there's no point in using latched mode if you have to disable it to create the comments.
